eclair-core/src/main/scala/fr/acinq/eclair/Setup.scala

```diff
@@ -270,17 +273,20 @@ class Setup(datadir: File,
           implicit val timeout = Timeout(30 seconds)
           new ElectrumEclairWallet(electrumWallet, nodeParams.chainHash)
       }
-      _ = wallet.getFinalAddress.map {
-        case address => logger.info(s"initial wallet address=$address")
-      }
+      _ = wallet.getFinalAddress.map(address => logger.info(s"initial wallet address=$address"))
       // do not change the name of this actor. it is used in the configuration to specify a custom bounded mailbox
 
-      backupHandler = system.actorOf(SimpleSupervisor.props(
-        BackupHandler.props(
-          nodeParams.db,
-          new File(chaindir, "eclair.sqlite.bak"),
-          if (config.hasPath("backup-notify-script")) Some(config.getString("backup-notify-script")) else None
-        ), "backuphandler", SupervisorStrategy.Resume))
+      backupHandler = if (config.getBoolean("enable-db-backup")) {
+        system.actorOf(SimpleSupervisor.props(
+          BackupHandler.props(
+            nodeParams.db,
+            new File(chaindir, "eclair.sqlite.bak"),
+            if (config.hasPath("backup-notify-script")) Some(config.getString("backup-notify-script")) else None
```

Comment:
Have you noticed that for the `backup-notify-script` parameter (which is also sensitive), we chose to not document it at all? Could have used the same pattern for `enable-db-backup`. And in that case, using the negative would make more sense...

Reply:
It is documented in `reference.conf`: `// override this with a script/exe that will be called everytime a new database backup has been created`...
I don't have a strong opinion honestly, let me know what you prefer :)

Reply:
Right 🤦‍♂️ . Nevermind.